src/main/resources/db/migration/V1_1__init_tables.sql

```diff
@@ -72,6 +72,20 @@ CREATE TABLE role_assignment_request(
 	CONSTRAINT role_assignment_request_pkey PRIMARY KEY (id)
 );
 
+CREATE TABLE audit_faults (
+	id int8 NOT NULL,
+	failed_at varchar NULL,
+	reason varchar NULL,
+	ccd_users varchar NULL,
+	request varchar NULL,
+	history varchar NULL,
+	live varchar NULL
+);
+
+
+create sequence AUDIT_ID_SEQ;
+ALTER TABLE audit_faults ALTER COLUMN id
+SET DEFAULT nextval('AUDIT_ID_SEQ');
```

Comment:
Why do we have role_assignment create table statements in this RAS batch job..This batch job is not supposed to create any table except this audit_faults